### ISSUE TITLE
Core DNS container won't start

### ISSUE THREAD

**Issue by CSnipper** [home-assistant/hassio-installer#87]
## The problem

Clean install newest Hassio image on OMV 4 or 5, computer: Intel I3, after install core_dns log:

> Listen: listen tcp :53: socket: permission denied

System log from Supervisor:

> 20-03-12 19:47:42 INFO (MainThread) [supervisor.dns] Start CoreDNS plugin
> 20-03-12 19:47:42 INFO (SyncWorker_15) [supervisor.docker.interface] Clean hassio_dns application
> 20-03-12 19:47:45 INFO (SyncWorker_15) [supervisor.docker.dns] Start DNS homeassistant/amd64-hassio-dns with version 1 - 172.30.32.3
> 20-03-12 19:48:02 WARNING (MainThread) [supervisor.tasks] Watchdog found a problem with CoreDNS plugin!
> 20-03-12 19:48:02 WARNING (MainThread) [supervisor.tasks] CoreDNS plugin is in fails state / Reset config

Effect: I can't install any addon. In every addon installed I have "Permission denied" errors

## Environment

- Home Assistant release with the issue: 
- Last working Home Assistant release (if known): 
- Operating environment (Hass.io/Docker/Windows/etc.): 
- Integration causing this issue: 
- Link to integration documentation on our website: 

## Problem-relevant `configuration.yaml`

```yaml

```

## Traceback/Error logs

```txt

```

**Comment by pvizeli** [comment]
I don't think that will work on a OMV. We start to minimize the support to Ubuntu and Debian. All other can work but if there are issues, we don't invest effort to solve it.

**Comment by CSnipper** [comment]
So, what system should I install on my PC for hassio?

**Comment by pvizeli** [comment]
In that case, yes. Debian work. Could be that is not the docker-ce version

**Comment by dezanche** [comment]
I have the same problem with aarch64-hassio-supervisor:214 on OMV with Docker CE (19.03.8), and it affects also hassio-cli.
Logs:
`20-04-06 17:33:56 WARNING (MainThread) [supervisor.tasks] Watchdog found a problem with CoreDNS plugin!
20-04-06 17:33:56 ERROR (MainThread) [supervisor.tasks] CoreDNS plugin is in fails state / Reset config
20-04-06 17:33:57 INFO (MainThread) [supervisor.dns] Start CoreDNS plugin
20-04-06 17:33:57 INFO (SyncWorker_3) [supervisor.docker.interface] Clean hassio_dns application
20-04-06 17:33:58 INFO (SyncWorker_3) [supervisor.docker.dns] Start DNS homeassistant/aarch64-hassio-dns with version 3 - 172.30.32.3
20-04-06 17:34:11 WARNING (MainThread) [supervisor.tasks] Watchdog found a problem with cli plugin!
20-04-06 17:34:11 INFO (MainThread) [supervisor.cli] Start cli plugin
20-04-06 17:34:11 INFO (SyncWorker_4) [supervisor.docker.interface] Clean hassio_cli application
20-04-06 17:34:13 INFO (SyncWorker_4) [supervisor.docker.cli] Start CLI homeassistant/aarch64-hassio-cli with version 24 - 172.30.32.4`
In the SSH terminal the DNS server is incorrectly set up:
`~ $ ha dns info
host: 172.30.32.3
locals: []
servers: []
version: "3"`
I tried everything I could find on the forums to fix the DNS problem but nothing worked (reverts automatically to the above).
Please help!

**Comment by dezanche** [comment]
> OMV is pure Debian, it is problem with docker I think.

what should I check specifically in docker?
after recent updates I now have 3 containers that will not run: hassio_dns, hassio_cli, and hassio_multicast
